doc/marian-integration.md

```diff
@@ -0,0 +1,88 @@
+# Marian Integration
+
+This document summarizes the minimal build instructions develop for the
+marian-code powering bergamot-translator.
+
+## Build Instructions
+
+```
+$ git clone https://github.com/browsermt/bergamot-translator
+$ cd bergamot-translator
+$ mkdir build
+$ cd build
+$ cmake .. -DCOMPILE_CUDA=off -DCMAKE_BUILD_TYPE=Release \
+     -DCOMPILE_DECODER_ONLY=on -DUSE_MKL=on -DCOMPILE_THREAD_VARIANT=on \
+     -DUSE_WASM_COMPATIBLE_BLAS=off -DCOMPILE_MAIN_EIGEN=off
```

Comment:
Why are we compiling stuff with `USE_MKL=on`?
And why `USE_WASM_COMPATIBLE_BLAS` is set to `off`?

Reply:
You're not the only customer here.  If Jerin can make this efficient enough, it should replace the current queuing implementation in `marian-decoder`.  

In any case, we should always benchmark against the best possible native performance.

Reply:
Okay. What is the goal of having this document? (PS: The instructions on how to compile bergamot-translator are already in top level README file).

Reply:
Well. I fixed this. You have one cmake option that can help you compile vanilla marian.